Clojure-Collections.md

```diff
@@ -0,0 +1,20 @@
+# Clojure - Collections
+One of Clojure's main positives is that it has quite a few different kinds of
```

Comment:
I try to keep each line around 80 characters long so that it's easy for me to read while editing. This is also what leads to the jumbling around of new lines (it makes inserting things difficult).